solutions/observability/streams/management/advanced.md

```diff
@@ -1,11 +1,40 @@
 ---
 applies_to:
-  serverless: preview
-  stack: preview 9.1
+  serverless: ga
+  stack: preview 9.1, ga 9.2
 navigation_title: Configure advanced settings
 ---
 # Configure advanced settings for streams [streams-advanced-settings]
 
-The **Advanced** tab on the **Manage stream** page shows the underlying configuration details of your stream. While Streams simplifies many configurations, it doesn't support modifying all pipelines and templates. From the **Advanced** tab, you can manually interact with the index or component templates or modify other ingest pipelines that used by the stream.
+The **Advanced** tab shows the underlying {{es}} configuration details and advanced configuration options for your stream.
 
-This UI is intended for advanced users.
\ No newline at end of file
+You can use the **Advanced** tab to manually configure the index or component templates or modify other ingest pipelines used by the stream.
+
+## Stream description
+
+% can we provide more information on what users should put here.
+
+Describe the data in the stream. AI features like system identification and significant events use this description when generating suggestions.
```

Comment:
Are system identification and significant events Elastic AI features? I'm not super familiar with the GenAI features in Observability, so forgive my ignorance if they are 😅 

```suggestion
Describe the data in the stream. Elastic AI features like system identification and significant events use this description when generating suggestions.
```